Subtester/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -70,12 +68,4 @@ android {
 // When using Unity Package Manager, this link will be wrong but still needed.
 **EXTERNAL_SOURCES**
 
-// IMPORTANT: If you're not installing through Unity Package Manager, remove the code below.
-// Symlink local sources. When using Unity's "Symlink local sources" checkbox,
-// The link will be wrong when using Unity Package Manager because of a bug in Unity.
-// That's not a problem if it points to a file that doesn't exist, but we need to add the path
-// that does exist here.
-// More info: https://forum.unity.com/threads/native-sources-inside-package-not-found-after-exporting-project-to-android.1355321/
-android.sourceSets.main.java.srcDirs += [
-    '**DIR_UNITYPROJECT**/../RevenueCat/Plugins/Android/'
-]
```

Comment:
This was causing a duplicated classes issue @tonidero, I removed it and it seems to be working fine